utils/chat.js

```diff
@@ -73,6 +81,7 @@ const processReplyMessage = async ({
     );
 
     baseMessage.reply_data = replyData;
+    baseMessage.quoted_message_id = replyMessageId;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `quoted_message_id` to `baseMessage` seems fine, but please make sure that this property is always available and valid when this function is called. If it's possible for `replyMessageId` to be `null` or `undefined`, you might want to add a check before assigning it to `baseMessage.quoted_message_id`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->